clarity.d.ts

```diff
@@ -266,6 +266,9 @@ interface IInstrumentationEventState {
 interface IJsErrorEventState extends IInstrumentationEventState {
   source: string;
   message: string;
+  stack: string;
```

Comment:
Since you updated the schema please update the schema version (I think it's found in core.ts). Rest looks good.